### PR TITLE
chore: fix batch denom examples

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Update example `batch_denom` to use a valid `batch_denom`.  

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/client/tx.go [hunk 1]
```diff
@@ -278,7 +278,7 @@ func TxSendCmd() *cobra.Command {
 Parameters:
   recipient: recipient address
   credits:   YAML encoded credit list. Note: numerical values must be written in strings.
-             eg: '[{batch_denom: "100/2", tradable_amount: "5", retired_amount: "0", retirement_location: "YY-ZZ 12345"}]'
+             eg: '[{batch_denom: "C01-20210101-20220101-001", tradable_amount: "5", retired_amount: "0", retirement_location: "YY-ZZ 12345"}]'
```
Comment (anilCSE) [inline x/ecocredit/client/tx.go:281]: nit: do we need to have validation to not accept `/` or any other special chars in the denom?

Comment (ryanchristo) [reply x/ecocredit/client/tx.go:281]: We already require the denom to conform to `FormatDenom` (i.e. `%s-%s-%s-%03d`). We will not accept `/` or any other special chars in the denom. This change here is only updating the examples in the CLI descriptions to do so.

Comment (ryanchristo) [reply x/ecocredit/client/tx.go:281]: We already require the denom to conform to `FormatDenom` (i.e. `%s-%s-%s-%03d`). We will not accept `/` or any other special chars in the denom. This change here is only updating the examples in the CLI descriptions.

Comment (aleem1314) [reply x/ecocredit/client/tx.go:281]: Looks like we are not using `ValidateDenom` in `MsgSend` and `MsgRetire`.

Comment (ryanchristo) [reply x/ecocredit/client/tx.go:281]: Opened #551.

**Review by aleem1314 — APPROVED** [review]

**Review by atheeshp — APPROVED** [review]

**Pull request merged**